pkg/auth/dependency/principal/password/provider.go

```diff
@@ -47,8 +43,7 @@ func newProvider(
 	reservedNameChecker *ReservedNameChecker,
 ) *providerImpl {
 	return &providerImpl{
-		sqlBuilder:   builder,
-		sqlExecutor:  executor,
+		store:        newStore(builder, executor),
```

Comment:
I think we should pass the store in arguments. This is more aligned with our convention.